third_party/blink/renderer/modules/global_privacy_control/navigator_global_privacy_control.idl

```diff
@@ -0,0 +1,9 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+[
+    ImplementedAs=NavigatorGlobalPrivacyControl
+] partial interface Navigator {
+    readonly attribute boolean globalPrivacyControl;
+};
```

Comment:
@bridiver do you know of a way to make the presence of an IDL attribute gated by a feature flag?